pallets/aleph/src/lib.rs

```diff
@@ -381,6 +451,51 @@ pub mod pallet {
 
             Ok(())
         }
+
+        // fix weight
+        #[pallet::call_index(3)]
+        #[pallet::weight(T::BlockWeights::get().max_block * Perbill::from_percent(10))]
+        /// Stores abft score
+        pub fn unsigned_submit_abft_score(
```

Comment:
When I try to send this ext as unsigned from azero.dev, it gives me `1012: Transaction is temporarily banned` - is it expected?

![image](https://github.com/user-attachments/assets/8b41fe66-9a92-4c98-b62e-c81186989d63)

Reply:
This is unsigned xt and you try to send is as signed, right?

Reply:
And this is different from `InvalidTransaction::Future` and `InvalidTransaction::Stale` since I can reproduce this errors by sending for future and past session respectively. The UI says in those scenarios  `1010: Invalid Transaction: Transaction will be valid in the future` and `1010: Invalid Transaction: Transaction is outdated`

Reply:
I click this button 
![image](https://github.com/user-attachments/assets/b07c10aa-f4c3-4faa-8ddc-6ed6e666992e)

Reply:
Let's leave testing for e2e phase